<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="15636" started-at="2020-03-30T16:09:13 BRT" finished-at="2020-03-30T16:09:29 BRT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="15636" started-at="2020-03-30T16:09:13 BRT" finished-at="2020-03-30T16:09:29 BRT">
      <class name="br.com.rsinet.hub.tdd.test.RealizarCadastro">
        <test-method status="PASS" signature="iniciaReport()[pri:0, instance:br.com.rsinet.hub.tdd.test.RealizarCadastro@5cdd8682]" name="iniciaReport" is-config="true" duration-ms="73" started-at="2020-03-30T16:09:13 BRT" finished-at="2020-03-30T16:09:13 BRT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iniciaReport -->
        <test-method status="PASS" signature="Inicializar()[pri:0, instance:br.com.rsinet.hub.tdd.test.RealizarCadastro@5cdd8682]" name="Inicializar" is-config="true" duration-ms="12602" started-at="2020-03-30T16:09:13 BRT" finished-at="2020-03-30T16:09:26 BRT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Inicializar -->
        <test-method status="FAIL" signature="cadastroComSucesso()[pri:0, instance:br.com.rsinet.hub.tdd.test.RealizarCadastro@5cdd8682]" name="cadastroComSucesso" duration-ms="2199" started-at="2020-03-30T16:09:26 BRT" finished-at="2020-03-30T16:09:28 BRT">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at br.com.rsinet.hub.tdd.test.RealizarCadastro.cadastroComSucesso(RealizarCadastro.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cadastroComSucesso -->
        <test-method status="PASS" signature="finalizaTest(org.testng.ITestResult)[pri:0, instance:br.com.rsinet.hub.tdd.test.RealizarCadastro@5cdd8682]" name="finalizaTest" is-config="true" duration-ms="373" started-at="2020-03-30T16:09:28 BRT" finished-at="2020-03-30T16:09:28 BRT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=cadastroComSucesso status=FAILURE method=RealizarCadastro.cadastroComSucesso()[pri:0, instance:br.com.rsinet.hub.tdd.test.RealizarCadastro@5cdd8682] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finalizaTest -->
        <test-method status="PASS" signature="finalizaExtent()[pri:0, instance:br.com.rsinet.hub.tdd.test.RealizarCadastro@5cdd8682]" name="finalizaExtent" is-config="true" duration-ms="365" started-at="2020-03-30T16:09:28 BRT" finished-at="2020-03-30T16:09:29 BRT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finalizaExtent -->
      </class> <!-- br.com.rsinet.hub.tdd.test.RealizarCadastro -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
